app/views/propertyrepresentation/revokeAgentProperties.scala.html

```diff
@@ -78,7 +78,7 @@ <h2 class="heading-small margin-top-0">@messages("propertyRepresentation.search.
             <div class="row">
                 <div class="column-one-half">
                     <div class="form-group" data-error="">
-                        <label class=" " for="address">
+                        <label class=" " for="Address">
```

Comment:
1. can you remove `class=" "` throughout to tidy up, pls
2. why did you uppercase `for="Address"`?